### PR TITLE
French Translation

### PR DESCRIPTION
**Pull request by Alfred-Neuman**
**When merged this pull request will:**
- _Describe what this pull request will do_
- _Each change in a separate line_

### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/) are read, understood and applied.
- Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.

### CODE REVIEW

File: addons/breathing/stringtable.xml [hunk 1]
```diff
@@ -2748,50 +2748,62 @@
         <Key ID="STR_KAT_Breathing_SETTING_AVAIL_OXYMASK">
             <English>Available Oxygen Masks</English>
             <Japanese>使用可能な酸素マスク</Japanese>
+            <French>Masques à oxygène disponibles</French>
         </Key>
         <Key ID="STR_KAT_Breathing_SETTING_AVAIL_OXYMASK_DISC">
             <English>Sets which goggles can act as oxygen masks</English>
             <Japanese>酸素マスクとして動作させたいゴーグルを指定します</Japanese>
+            <French>Spécifiez les lunettes que vous souhaitez utiliser comme masque à oxygène.</French>
```
Comment (Fanrod-Bones) [inline addons/breathing/stringtable.xml]: Spécifiez les masques que vous souhaitez utiliser comme masque à oxygène.

(Here we make reference to the items in the arsenal, imo "masques" is less confusing)

File: addons/breathing/stringtable.xml [hunk 2]
```diff
@@ -2748,50 +2748,62 @@
...
         </Key>
         <Key ID="STR_KAT_Breathing_PersonalOxygen_Empty">
             <English>Oxygen Tank Empty</English>
             <Japanese>酸素ボンベは空です</Japanese>
+            <French>Réservoir d'oxygène vide</French>
         </Key>
         <Key ID="STR_KAT_Breathing_AttachOxygenVehicle">
             <English>Connect to Aircraft Oxygen</English>
             <Japanese>航空機酸素供給機に接続</Japanese>
+            <French>Connexion à l'oxygène de l'avion</French>
```
Comment (Fanrod-Bones) [inline addons/breathing/stringtable.xml]: Connexion à l'oxygène de l'aéronef

(reference to both hélicoptère and avion) (Will mostly be helicopter I think tho)

File: addons/breathing/stringtable.xml [hunk 3]
```diff
@@ -2748,50 +2748,62 @@
...
         </Key>
         <Key ID="STR_KAT_Breathing_VehicleOxygen_Affirm">
             <English>Attached to Aircraft Oxygen</English>
             <Japanese>航空機酸素供給機に接続済</Japanese>
+            <French>Attaché à l'oxygène de l'avion</French>
```
Comment (Fanrod-Bones) [inline addons/breathing/stringtable.xml]: Attaché à l'oxygène de l'aéronef

File: addons/chemical/stringtable.xml [hunk 4]
```diff
@@ -799,9 +799,11 @@
         </Key>
         <Key ID="STR_KAT_Chemical_SETTING_dropWeaponChance">
             <English>Drop Weapon Chance</English>
+            <French>Chance de larguer l'arme</French>
```
Comment (Fanrod-Bones) [inline addons/chemical/stringtable.xml]: Chance de lâcher l'arme

File: addons/circulation/stringtable.xml [hunk 5]
```diff
@@ -2593,6 +2593,7 @@
         <Key ID="STR_KAT_Circulation_ChangeCPRDevice">
             <English>Change Monitoring Device</English>
             <Japanese>モニタリングする装置を変更</Japanese>
+            <French>Changer le Dispositif de surveillance</French>
```
Comment (Fanrod-Bones) [inline addons/circulation/stringtable.xml]: Changer le dispositif de surveillance

(D to d)

**Review by Fanrod-Bones — CHANGES_REQUESTED** [review]
Small adjustments but otherwise as always all the rest lgtm

We should get in touch to redo the french guide btw, I am not sure the guy that did it in the past update it anymore !

**Comment by Alfred-Neuman** [comment]
@Fanrod-Bones If you like. 

Anyway, thanks for proofreading.
I've got my head in the game and mistakes happen fast.

**Comment by MiszczuZPolski** [comment]
Get me on discord so i can give you contributor role

**Pull request merged**